.github/workflows/ci.yaml

```diff
@@ -130,27 +130,32 @@ jobs:
               run: choco upgrade llvm
             - name: Setup headless display
               uses: pyvista/setup-headless-display-action@v1
-            - uses: ./.github/actions/install-nodejs
```

Comment:
I think this action is unused now and can get removed.

That action did some extra steps. Is it save to remove those? E.g. install node-gyp on Windows?

Reply:
done